### PR TITLE
Fixes#251 Updates EhealthAfrica sample

### PR DESCRIPTION
**Pull request by ParthS007**
Updates eHealth Africa sample
Live Link : http://opev-webgen-dev.herokuapp.com/live/preview/parth1989shandilya@gmail.com/GlobalHealthSecurity%E2%80%93FocusAfrica/

@mariobehling  @geekyd  Please review

**I will add it to overview site after merging of this PR**

### CODE REVIEW

**Comment by geekyd** [comment]
@ParthS007 The provided link is not working. Can you update the link?

**Comment by ParthS007** [comment]
@geekyd  Done !

**Comment by geekyd** [comment]
@mariobehling @hpdang LGTM!

File: sample/EhealthAfrica/event [hunk 1]
```diff
@@ -1,60 +1,59 @@
...
+                    "link": "https://www.facebook.com/EHealthAfrica", 
+                    "name": "Facebook"
         }, 
         {
-            "id": 361, 
-            "link": "https://twitter.com/eHealth_africa", 
-            "name": "Twitter"
+                    "id": 361, 
+                    "link": "https://twitter.com/eHealth_africa", 
+                    "name": "Twitter"
```
Comment (agbilotia1998) [inline sample/EhealthAfrica/event]: @ParthS007 this indentation is incorrect.

Comment (ParthS007) [reply sample/EhealthAfrica/event]: Thanks @agbilotia1998 , will do the changes

**Comment by ParthS007** [comment]
@mariobehling @SaptakS @geekyd  @agbilotia1998 Please review

**Pull request merged**